crates/indexer/src/models/token_models/token_utils.rs

```diff
@@ -48,6 +52,10 @@ impl TokenDataIdType {
     pub fn get_collection_data_id_hash(&self) -> String {
         CollectionDataIdType::new(self.creator.clone(), self.collection.clone()).to_hash()
     }
+
+    pub fn get_collection_id(&self) -> String {
```

Comment:
Why do we need both `get_collection_id` and `get_collection_data_id_hash`?

Reply:
get_collection_id is b/c v2 actually has addresses now so we should have standardized pattern. Still have to keep both for backwards compatibility.